### PR TITLE
Update some screenshots of the launcher in guides/getting-minecraft-game-output-log

### PR DESCRIPTION
**Pull request by Southpaw1496**
The launcher has changed since the original screenshots were taken so I updated the first three. The others I left alone because that part of the interface has remained unchanged and there is a bug preventing the logs from displaying correctly on macOS.

### CODE REVIEW

File: _help/common-problems-and-fixes/java/index.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Account Migration issues
+If it is unclear, establish whether the user is trying to migrate from a Legacy account to a Mojang account or a Mojang account to a Microsoft account. If they are trying to migrate a Legacy account, inform them that Legacy -> Mojang migration is currently unavailable but they can still log into the launcher. If they are trying to migrate to a Microsoft account, explain that account migration has not yet started, and link them to the account migration FAQ page at https://help.minecraft.net/hc/en-us/articles/360050865492 (Absol Factoid: `account/microsoft`)
+
+## If you have no ideas
+Run HiJackThis and Adwcleaner to check for incompatible software. For networking issues, have them try on a mobile hotspot and change their DNS (https://1.1.1.1/dns/). Google and https://bugs.mojang.com are also good places to search for issues.
```
Comment (urielsalis) [inline _help/common-problems-and-fixes/java/index.markdown:43]: I dont recommend telling people to run HJT without supervision, they can tell it to clean stuff and break stuff

Comment (Southpaw1496) [reply _help/common-problems-and-fixes/java/index.markdown:43]: Could we ask them to run it using our guide specifically? That would be no less supervision than normal

Comment (urielsalis) [reply _help/common-problems-and-fixes/java/index.markdown:43]: supervision means we tell them to run it, they give us the results and we tell them what to do

Comment (Southpaw1496) [reply _help/common-problems-and-fixes/java/index.markdown:43]: Wait a minute, the common problems and fixes file isn't meant to be in this PR, it's meant to be in #92 

sigh

Should I just delete it or will it break git?

**Pull request closed without merging**